pandas/tests/indexes/test_multi.py

```diff
@@ -1425,8 +1448,6 @@ def test_equals_multi(self):
 
         self.assertFalse(self.index.equals(self.index[:-1]))
 
-        self.assertTrue(self.index.equals(self.index._tuple_index))
```

Comment:
hmm, I remember that. Though [25] should not be True then. (though I guess its an index, and NOT a list-like). But its NOT an MI either....